pandas/core/tools/datetimes.py

```diff
@@ -577,14 +577,13 @@ def to_datetime(
 
     Parameters
     ----------
-    arg : int, float, str, datetime, list, tuple, 1-d array, Series
-           or DataFrame/dict-like
-
+    arg : int, float, str, datetime, list, tuple, 1-d array, Series DataFrame/dict-like
+        The object to convert to a datetime.
     errors : {'ignore', 'raise', 'coerce'}, default 'raise'
-
-        - If 'raise', then invalid parsing will raise an exception
-        - If 'coerce', then invalid parsing will be set as NaT
-        - If 'ignore', then invalid parsing will return the input
+        Behaves as:
```

Comment:
Was this required to get validation to work?

Reply:
Yes, otherwise it raises error PR08, that the description should start with a capital letter.

Reply:
You can remove all this. The new version of the script doesn't fail if the description starts with a list. And we'll move to the new version of the script soon.